main.go

```diff
@@ -60,6 +60,7 @@ func main() {
 	var (
 		broker   = flag.String("broker", "tcp://localhost:1883", "MQTT broker endpoint as scheme://host:port")
 		topic    = flag.String("topic", "/test", "MQTT topic for outgoing messages")
+		payload  = flag.String("payload", "", "MQTT message payload. If empty, then payload is generated based on the size parameter")
```

Comment:
Please add an example running the tool with a custom payload in README.

Reply:
done :)